### DISCUSSION TITLE
Formatting Changes and Convenience Edits

### DISCUSSION THREAD

**Discussion by Gandalf47** [naturalcrit/homebrewery#3247] [Ideas]
The first thing I wanted to address is how thankful I am for this website. It has helped so much, and I have enjoyed using it to a degree that I can't even express in text. Suffice it to say that I believe that the Homebrewery is one of the best things that has happened to the homebrew community.

I have some suggestions to make, however, regarding the format of the Class Features feature. First, in classic D&D content, the letters inside the parentheses for the equipment are italicized, not the whole thing. I know it sound incredibly stupid to mention it, but it's something I noticed. 

The second thing I'd like to mention are the filler stuff that comes in the monster stat block and other such presets. It would be helpful if they had stats that you were more likely to see on an actual monster; for example, instead of saying it's 2037 XP, say its 450 XP.

**Comment by 5e-Cleric** [comment]
Regarding your first comment, this is from just right now in the tool:

<img width="295" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/06eac703-b25e-4618-8bf4-75950094ee72">

They are not italicized, just the letter.

About the second point, well, we could change the random examples to include actual stat blocks such as a zombie, but those statblocks can be seen in other books, so most people will not write them down, instead, will write their own, and will therefore change every number and letter, the example is good as is, but we'll give a though to it.

We're happy to receive the feedback though.

**Reply by Gandalf47** [reply]
I think I might have mistyped. What I meant was that the parentheses are not italicized in classic D&D content.

**Reply by 5e-Cleric** [reply]
That is a good point

**Comment by Gazook89** [comment]
Going to close this discussion since the most fixable issue was fixed awhile ago and the other issue is closer to “won’t fix”.
